app/src/main/java/net/aiscope/gdd_app/ui/capture/ZoomController.kt

```diff
@@ -98,4 +104,15 @@ class ZoomController(
         handler.removeCallbacks(hideZoomLevelSeekBarRunnable)
         cameraZoomLevel.isVisible = true
     }
+
+    private fun onZoomChanged(zoom: Float) {
+        // it is assumed that when the value provided here is equal to 1.0
+        // then the image is not zoomed
+        // and also that zoom cannot be less than 1.0
+        onZoomChangedListener?.onZoomChanged(zoom > 1.0f);
```

Comment:
May I suggest using a named argument here for more clarity? Also no semicolon, please :P
```suggestion
        onZoomChangedListener?.onZoomChanged(isZoomed = zoom > 1.0f)
```